diff --git a/qtbase/src/corelib/Qt6AndroidMacros.cmake b/qtbase/src/corelib/Qt6AndroidMacros.cmake
index 4f6eadb..0b40c6a 100644
--- a/qtbase/src/corelib/Qt6AndroidMacros.cmake
+++ b/qtbase/src/corelib/Qt6AndroidMacros.cmake
@@ -264,7 +264,7 @@ function(qt6_android_generate_deployment_settings target)
 
     # base location of stdlibc++, will be suffixed by androiddeploy qt
     # Sysroot is set by Android toolchain file and is composed of ANDROID_TOOLCHAIN_ROOT.
-    set(android_ndk_stdlib_base_path "${CMAKE_SYSROOT}/usr/lib/")
+    set(android_ndk_stdlib_base_path "${CMAKE_ANDROID_NDK_TOOLCHAIN_UNIFIED}/sysroot/usr/lib/")
     string(APPEND file_contents
         "   \"stdcpp-path\": \"${android_ndk_stdlib_base_path}\"\n")
 
