diff --git a/CMakeLists.txt b/CMakeLists.txt
index b412dc7..b2fb7fa 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -9,7 +9,7 @@ set(INSTALL_BIN_DIR "${CMAKE_INSTALL_PREFIX}/bin" CACHE PATH "Installation direc
 set(INSTALL_LIB_DIR "${CMAKE_INSTALL_PREFIX}/lib" CACHE PATH "Installation directory for libraries")
 set(INSTALL_INC_DIR "${CMAKE_INSTALL_PREFIX}/include" CACHE PATH "Installation directory for headers")
 set(INSTALL_MAN_DIR "${CMAKE_INSTALL_PREFIX}/share/man" CACHE PATH "Installation directory for manual pages")
-set(INSTALL_PKGCONFIG_DIR "${CMAKE_INSTALL_PREFIX}/share/pkgconfig" CACHE PATH "Installation directory for pkgconfig (.pc) files")
+set(INSTALL_PKGCONFIG_DIR "${CMAKE_INSTALL_PREFIX}/lib/pkgconfig" CACHE PATH "Installation directory for pkgconfig (.pc) files")
 
 include(CheckTypeSize)
 include(CheckFunctionExists)
@@ -171,7 +171,13 @@ if(UNIX)
    endif()
 elseif(BUILD_SHARED_LIBS AND WIN32)
     # Creates zlib1.dll when building shared library version
-    set_target_properties(zlib PROPERTIES SUFFIX "1.dll")
+    # set_target_properties(zlib PROPERTIES SUFFIX "1.dll")
+
+    # The pre-built libraries for fedora name the library libz.dll.a,
+    # so do the same here.  Helpful, as several of the libraries being
+    # built for MythTV have a hard-coded -lz in their list of
+    # libraries to link.
+    set_target_properties(zlib zlibstatic PROPERTIES OUTPUT_NAME z)
 endif()
 
 if(NOT SKIP_INSTALL_LIBRARIES AND NOT SKIP_INSTALL_ALL )
diff --git a/zlib.map b/zlib.map
index b330b60..9600ca1 100644
--- a/zlib.map
+++ b/zlib.map
@@ -15,7 +15,6 @@ ZLIB_1.2.0 {
     zcfree;
     z_errmsg;
     gz_error;
-    gz_intmax;
     _*;
 };
 
