diff --git mythplugins/mythgame/i18n/mythgame_it.ts mythplugins/mythgame/i18n/mythgame_it.ts
index 53ea8adf3f..f397ed8227 100644
--- mythplugins/mythgame/i18n/mythgame_it.ts
+++ mythplugins/mythgame/i18n/mythgame_it.ts
@@ -69,7 +69,7 @@
     <message>
         <location filename="../mythgame/gamesettings.cpp" line="170"/>
         <source>MythGame Settings -- General</source>
-        <translation type="unfinished">Impostazioni MythGame - generale</translation>
+        <translation>Impostazioni MythGame - generale</translation>
     </message>
 </context>
 <context>
@@ -192,24 +192,24 @@ Rimuoverlo dal database?</translation>
     <message>
         <location filename="../mythgame/gamesettings.cpp" line="348"/>
         <source>Game Players</source>
-        <translation type="unfinished">Giocatori</translation>
+        <translation>Giocatori</translation>
     </message>
     <message>
         <location filename="../mythgame/gamesettings.cpp" line="355"/>
         <source>(New Game Player)</source>
-        <translation type="unfinished">(nuovo giocatore)</translation>
+        <translation>(nuovo giocatore)</translation>
     </message>
     <message>
         <location filename="../mythgame/gamesettings.cpp" line="361"/>
         <source>%1 (%2)</source>
         <comment>Game player/emulator display</comment>
         <extracomment>%1 is the player/emulator name, %2 is the type of player/emulator</extracomment>
-        <translation type="unfinished">%1 (%2)</translation>
+        <translation>%1 (%2)</translation>
     </message>
     <message>
         <location filename="../mythgame/gamesettings.cpp" line="393"/>
         <source>Player Name</source>
-        <translation type="unfinished">Nome giocatore</translation>
+        <translation>Nome giocatore</translation>
     </message>
 </context>
 <context>
@@ -844,12 +844,12 @@ di volerlo fare?</translation>
     <message>
         <location filename="themestrings.h" line="108"/>
         <source>Type</source>
-        <translation type="unfinished">Tipo</translation>
+        <translation>Tipo</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="111"/>
         <source>Year: %1</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
     </message>
     <message>
         <location filename="themestrings.h" line="112"/>
@@ -924,7 +924,7 @@ di volerlo fare?</translation>
     <message>
         <location filename="themestrings.h" line="90"/>
         <source>Rompath:</source>
-        <translation type="unfinished">Percorso Rom:</translation>
+        <translation>Percorso Rom:</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="103"/>
@@ -970,57 +970,57 @@ di volerlo fare?</translation>
         <location filename="themestrings.h" line="5"/>
         <source>%00x00| - %%&quot;|SUBTITLE|&quot;
 %%|YEARSTARS| - %%DESCRIPTION%</source>
-        <translation type="unfinished">%00x00| - %%&quot;|SUBTITLE|&quot;
+        <translation>%00x00| - %%&quot;|SUBTITLE|&quot;
 %%|YEARSTARS| - %%DESCRIPTION%</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="7"/>
         <source>%Published by |publisher|%%, |YEAR%</source>
-        <translation type="unfinished">%Pubblicato da|publisher|%%, |YEAR%</translation>
+        <translation>%Pubblicato da|publisher|%%, |YEAR%</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="8"/>
         <source>%&quot;|SUBTITLE|&quot;
 
 %%DESCRIPTION%</source>
-        <translation type="unfinished">%&quot;|SUBTITLE|&quot;
+        <translation>%&quot;|SUBTITLE|&quot;
 
 %%DESCRIPTION%</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="9"/>
         <source>%cast%</source>
-        <translation type="unfinished">%cast%</translation>
+        <translation>%cast%</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="10"/>
         <source>%playcount% times</source>
-        <translation type="unfinished">%playcount% volte</translation>
+        <translation>%playcount% volte</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="11"/>
         <source>%romname%%, |(rompath)%</source>
-        <translation type="unfinished">%romname%%, |(rompath)%</translation>
+        <translation>%romname%%, |(rompath)%</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="13"/>
         <source>Add game to Favorites:</source>
-        <translation type="unfinished">Aggiungi gioco ai preferiti:</translation>
+        <translation>Aggiungi gioco ai preferiti:</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="14"/>
         <source>Add games to your</source>
-        <translation type="unfinished">Aggiungi giochi al tuo</translation>
+        <translation>Aggiungi giochi al tuo</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="16"/>
         <source>Add to favorites:</source>
-        <translation type="unfinished">Aggiungi ai preferiti:</translation>
+        <translation>Aggiungi ai preferiti:</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="19"/>
         <source>CRC</source>
-        <translation type="unfinished">CRC</translation>
+        <translation>CRC</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="21"/>
@@ -1030,57 +1030,57 @@ di volerlo fare?</translation>
     <message>
         <location filename="themestrings.h" line="23"/>
         <source>Clear data of all your games</source>
-        <translation type="unfinished">Cancella dati di tutti i giochi</translation>
+        <translation>Cancella dati di tutti i giochi</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="24"/>
         <source>Clear your game</source>
-        <translation type="unfinished">Cancella il tuo gioco</translation>
+        <translation>Cancella il tuo gioco</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="29"/>
         <source>Configure game emulators</source>
-        <translation type="unfinished">Configura emulatori giochi</translation>
+        <translation>Configura emulatori giochi</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="30"/>
         <source>Configure your game</source>
-        <translation type="unfinished">Configura gioco</translation>
+        <translation>Configura gioco</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="31"/>
         <source>Country</source>
-        <translation type="unfinished">Paese</translation>
+        <translation>Paese</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="34"/>
         <source>Coverart</source>
-        <translation type="unfinished">Copertina</translation>
+        <translation>Copertina</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="35"/>
         <source>Coverart:</source>
-        <translation type="unfinished">Copertina:</translation>
+        <translation>Copertina:</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="36"/>
         <source>Customize meta data of your games</source>
-        <translation type="unfinished">Personalizza metadati dei giochi</translation>
+        <translation>Personalizza metadati dei giochi</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="37"/>
         <source>Data</source>
-        <translation type="unfinished">Dati</translation>
+        <translation>Dati</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="39"/>
         <source>Description: %1</source>
-        <translation type="unfinished">Descrizione: %1</translation>
+        <translation>Descrizione: %1</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="40"/>
         <source>Detailed information about your game</source>
-        <translation type="unfinished">Informazioni dettagliate sul gioco</translation>
+        <translation>Informazioni dettagliate sul gioco</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="43"/>
@@ -1090,57 +1090,57 @@ di volerlo fare?</translation>
     <message>
         <location filename="themestrings.h" line="46"/>
         <source>Fanart</source>
-        <translation type="unfinished">Fanart</translation>
+        <translation>Fanart</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="50"/>
         <source>File:</source>
-        <translation type="unfinished">File:</translation>
+        <translation>File:</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="51"/>
         <source>Filename</source>
-        <translation type="unfinished">Nome file</translation>
+        <translation>Nome file</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="54"/>
         <source>GAMES</source>
-        <translation type="unfinished">GIOCHI</translation>
+        <translation>GIOCHI</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="58"/>
         <source>Game Type:</source>
-        <translation type="unfinished">Tipo gioco:</translation>
+        <translation>Tipo gioco:</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="59"/>
         <source>Game details</source>
-        <translation type="unfinished">Dettagli gioco</translation>
+        <translation>Dettagli gioco</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="60"/>
         <source>Game folders and general</source>
-        <translation type="unfinished">Cartelle gioco e generale</translation>
+        <translation>Cartelle gioco e generale</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="62"/>
         <source>Game folders and general settings</source>
-        <translation type="unfinished">Cartelle gioco e impostazioni generali</translation>
+        <translation>Cartelle gioco e impostazioni generali</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="64"/>
         <source>Game plot:</source>
-        <translation type="unfinished">Trama gioco:</translation>
+        <translation>Trama gioco:</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="70"/>
         <source>Library</source>
-        <translation type="unfinished">Biblioteca</translation>
+        <translation>Biblioteca</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="71"/>
         <source>Metadata</source>
-        <translation type="unfinished">Metadati</translation>
+        <translation>Metadati</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="72"/>
@@ -1150,92 +1150,92 @@ di volerlo fare?</translation>
     <message>
         <location filename="themestrings.h" line="73"/>
         <source>Name:</source>
-        <translation type="unfinished">Nome:</translation>
+        <translation>Nome:</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="74"/>
         <source>Navigation</source>
-        <translation type="unfinished">Navigazione</translation>
+        <translation>Navigazione</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="77"/>
         <source>Path:</source>
-        <translation type="unfinished">Percorso:</translation>
+        <translation>Percorso:</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="80"/>
         <source>Players</source>
-        <translation type="unfinished">Giocatori</translation>
+        <translation>Giocatori</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="82"/>
         <source>Publisher</source>
-        <translation type="unfinished">Editore</translation>
+        <translation>Editore</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="86"/>
         <source>ROM path</source>
-        <translation type="unfinished">Percorso ROM</translation>
+        <translation>Percorso ROM</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="88"/>
         <source>ROM:</source>
-        <translation type="unfinished">ROM:</translation>
+        <translation>ROM:</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="91"/>
         <source>Save</source>
-        <translation type="unfinished">Salva</translation>
+        <translation>Salva</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="93"/>
         <source>Screenshot</source>
-        <translation type="unfinished">Schermata</translation>
+        <translation>Schermata</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="95"/>
         <source>Search</source>
-        <translation type="unfinished">Cerca</translation>
+        <translation>Cerca</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="96"/>
         <source>Search Boxart</source>
-        <translation type="unfinished">Cerca boxart</translation>
+        <translation>Cerca boxart</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="97"/>
         <source>Search Fanart</source>
-        <translation type="unfinished">Cerca fanart</translation>
+        <translation>Cerca fanart</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="98"/>
         <source>Search Screenshot</source>
-        <translation type="unfinished">Cerca schermata</translation>
+        <translation>Cerca schermata</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="99"/>
         <source>Select Game</source>
-        <translation type="unfinished">Seleziona gioco</translation>
+        <translation>Seleziona gioco</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="100"/>
         <source>Settings</source>
-        <translation type="unfinished">Impostazioni</translation>
+        <translation>Impostazioni</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="101"/>
         <source>Starring:</source>
-        <translation type="unfinished">Protagonista:</translation>
+        <translation>Protagonista:</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="102"/>
         <source>System</source>
-        <translation type="unfinished">Sistema</translation>
+        <translation>Sistema</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="106"/>
         <source>Systems:</source>
-        <translation type="unfinished">Sistemi:</translation>
+        <translation>Sistemi:</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="107"/>
@@ -1245,17 +1245,17 @@ di volerlo fare?</translation>
     <message>
         <location filename="themestrings.h" line="113"/>
         <source>_</source>
-        <translation type="unfinished">_</translation>
+        <translation>_</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="114"/>
         <source>genre:</source>
-        <translation type="unfinished">genere:</translation>
+        <translation>genere:</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="115"/>
         <source>system:</source>
-        <translation type="unfinished">sistema:</translation>
+        <translation>sistema:</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="33"/>
@@ -1360,7 +1360,7 @@ di volerlo fare?</translation>
     <message>
         <location filename="themestrings.h" line="92"/>
         <source>Scan for Games</source>
-        <translation>Scansione per giochi</translation>
+        <translation>Cerca giochi</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="15"/>
@@ -1370,7 +1370,7 @@ di volerlo fare?</translation>
     <message>
         <location filename="themestrings.h" line="22"/>
         <source>Clear Game Data</source>
-        <translation>Cancellare dati gioco</translation>
+        <translation>Cancella dati gioco</translation>
     </message>
     <message>
         <location filename="themestrings.h" line="44"/>
